### PR TITLE
Add kinzhi to sig-docs-zh-reviews

### PR DESCRIPTION
**Pull request by kinzhi**
I want to join the sig-docs-zh-reviews team.

I have met the [requirements for reviewer](https://github.com/kubernetes/community/blob/master/community-membership.md#reviewer):

- Member of Kubernetes for 3+ months
- 170+ PRs merged: https://github.com/kubernetes/website/pulls?q=is%3Apr+is%3Amerged+author%3Akinzhi
- 10+ PRs reviewed:https://github.com/kubernetes/website/pulls?q=is%3Apr+reviewed-by%3Akinzhi

Thanks

### CODE REVIEW

File: config/kubernetes/sig-docs/teams.yaml [hunk 1]
```diff
@@ -288,6 +288,7 @@ teams:
     - Sea-n
     - tanjunchen
     - tengqm
+    - kinzhi
```
Comment (tengqm) [inline config/kubernetes/sig-docs/teams.yaml:291]: You will have to make sure the list is still sorted.

Comment (kinzhi) [reply config/kubernetes/sig-docs/teams.yaml]: It has been adjusted, thank you

**Comment by ydFu** [comment]
@kinzhi  is an active contributor to localization in zh-cn, looking forward to your continued contribution

**Comment by kinzhi** [comment]
> @kinzhi is an active contributor to localization in zh-cn, looking forward to your continued contribution

Thank you, I will contribute more actively to the community.

**Pull request merged**